src/app/shared/utilities/view.utility.ts

```diff
@@ -160,21 +160,47 @@ const equals = (filterOne: Filter, filterTwo: Filter): boolean => {
   return filterTwo ? filterOne.field === filterTwo.field && filterOne.value === filterTwo.value : true;
 };
 
+/**
+ * Traverse object by path returning value else undefined.
+ *
+ * {
+ *   'trainee': {
+ *     'label': 'Name of organization'
+ *   }
+ * }
+ * i.e. `trainee.label` returns 'Name of organizarion'
+ *
+ * @param doc solr document or any JSON object
+ * @param path dot notation path
+ * @returns value at path
+ */
+const getValueByPath = (doc: any, path: string): string | undefined => {
+  let pathValue;
+  path.split('.').forEach((p: string) => {
+    pathValue = pathValue ? pathValue[p] : doc[p];
+  });
+  return pathValue;
+};
+
 const getResourcesPage = (resources: any[], sort: Sort[], page: SdrPage): any[] => {
-  let sorted = [].concat(resources);
+  let sorted = [].concat(resources); 
```

Comment:
Has a trailing space at the end of line 186.

Reply:
Good catch. Will correct.